tests/integration/telemetry/stackdriver/testdata/security_authz_dry_run/server_access_log_allow_no_policy.json.tmpl

```diff
@@ -19,7 +19,7 @@
       "source_namespace": "{{ .EchoNamespace }}",
       "destination_service_name": "srv",
       "destination_version": "v1",
-      "log_sampled": "true",
+      "log_sampled": "false",
```

Comment:
What's the reason for the change in value here?

Reply:
This is a part of the removal of the "access logging policy". I discussed this briefly with you on chat.

I don't see any real purpose for this field/filter. We just set this field, its not _actually_ sampled as far as I can tell -- the `log_sampled=false` requests still go to the real SD backend(!).

If we do want it still, we can add it as an environment var like we did with the AUDIT mode, but it doesn't seem useful. From my (imperfect) research, it has no usage, nor can I come up with even theoretical ideas on how it could be useful.